automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureRuntimeSpec.scala

```diff
@@ -95,6 +95,55 @@ class AzureRuntimeSpec
         )
         _ = monitorCreateResult.getStatus() shouldBe ClusterStatus.RUNNING
 
+        // Now stop the runtime
+        _ <- IO(runtimeClient.stopRuntimeV2(workspaceId, runtimeName.asString))
+
+        callGetRuntime = IO(runtimeClient.getAzureRuntime(workspaceId, runtimeName.asString))
+        stoppingRuntimeResponse <- callGetRuntime
+        _ <- loggerIO.info(s"stopping get runtime response ${stoppingRuntimeResponse}")
+        _ = stoppingRuntimeResponse.getStatus shouldBe ClusterStatus.STOPPING
+
+        _ <- loggerIO.info(
+          s"AzureRuntimeSpec: runtime ${workspaceId}/${runtimeName.asString} in stopping status detected"
+        )
+
+        // Verify the runtime eventually becomes Stopped
+        monitorStopResult <- streamUntilDoneOrTimeout(
+          callGetRuntime,
+          120,
+          10 seconds,
+          s"AzureRuntimeSpec: runtime ${workspaceId}/${runtimeName.asString} did not stop after 20 minutes"
```

Comment:
this can probably be a lower number, I think it usually takes <5 to stop, so I'd maybe cap it at 10